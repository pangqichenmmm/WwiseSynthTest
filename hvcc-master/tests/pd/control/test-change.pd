#N canvas 643 382 317 278 10;
#X obj 34 190 change 6.5;
#X msg 109 120 set \$1;
#X obj 34 230 print;
#X obj 109 190 change;
#X msg 109 90 5;
#X msg 49 93 -100;
#X obj 34 30 loadbang;
#X obj 34 60 t b b b b b b;
#X msg 34 132 230 2;
#X connect 0 0 2 0;
#X connect 1 0 3 0;
#X connect 3 0 2 0;
#X connect 4 0 1 0;
#X connect 5 0 0 0;
#X connect 6 0 7 0;
#X connect 7 0 8 0;
#X connect 7 1 5 0;
#X connect 7 2 5 0;
#X connect 7 3 0 0;
#X connect 7 4 3 0;
#X connect 7 5 4 0;
#X connect 8 0 0 0;
