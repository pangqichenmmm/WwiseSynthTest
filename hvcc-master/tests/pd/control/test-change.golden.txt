[@ 0.000] print: 5
[@ 0.000] print: 6.5
[@ 0.000] print: -100
[@ 0.000] print: 230
